src/decs.d.ts

```diff
@@ -0,0 +1 @@
+declare module "react-stack-grid"
```

Comment:
I think this file may not be necessary. Removed the `isolated-modules` flag in the `tsconfig`